src/heifimage.cpp

```diff
@@ -0,0 +1,76 @@
+#define DEBUG
```

Comment:
Ups, it looks like this should not be here.

Reply:
Removed, it was there by mistake.